src/_nebari/provider/cloud/google_cloud.py

```diff
@@ -57,7 +57,7 @@ def kubernetes_versions(region: str) -> List[str]:
     credentials, project_id = load_credentials()
     client = container_v1.ClusterManagerClient(credentials=credentials)
     response = client.get_server_config(
-        name=f"projects/{project_id}/locations/{region}"
+        name=f"projects/{project_id}/locations/{region}", timeout=300
```

Comment:
I was running into GRPC timeouts using the default 20s context timeout, so I set it to 5 minutes